lib/widgets/subscription_list.dart

```diff
@@ -210,36 +210,39 @@ class SubscriptionItem extends StatelessWidget {
             MessageListPage.buildRoute(context: context,
               narrow: StreamNarrow(subscription.streamId)));
         },
-        child: Row(crossAxisAlignment: CrossAxisAlignment.center, children: [
-          const SizedBox(width: 16),
-          Padding(
-            padding: const EdgeInsets.symmetric(vertical: 11),
-            child: Icon(size: 18, color: swatch.iconOnPlainBackground,
-              iconDataForStream(subscription))),
-          const SizedBox(width: 5),
-          Expanded(
-            child: Padding(
-              padding: const EdgeInsets.symmetric(vertical: 10),
-              // TODO(design): unclear whether bold text is applied to all subscriptions
-              //   or only those with unreads:
-              //   https://github.com/zulip/zulip-flutter/pull/397#pullrequestreview-1742524205
-              child: Text(
-                style: const TextStyle(
-                  fontSize: 18,
-                  height: (20 / 18),
-                  // TODO(#95) need dark-theme color
-                  color: Color(0xFF262626),
-                ).merge(weightVariableTextStyle(context,
-                    wght: hasUnreads ? 600 : null)),
-                maxLines: 1,
-                overflow: TextOverflow.ellipsis,
-                subscription.name))),
-          if (unreadCount > 0) ...[
-            const SizedBox(width: 12),
-            // TODO(#384) show @-mention indicator when it applies
-            UnreadCountBadge(count: unreadCount, backgroundColor: swatch, bold: true),
-          ],
-          const SizedBox(width: 16),
-        ])));
+        child: Opacity(
```

Comment:
Instead of `Opacity` wrapping the whole `Row`, let's make it wrap only the icon and the stream name. This will be helpful when there is a dot marker added through #714.